descheduler/templates/cronjob.yaml

```diff
@@ -12,13 +12,24 @@ spec:
   jobTemplate:
     metadata:
       labels:
+{{ if .Values.labels }}{{ toYaml .Values.labels | indent 8 }}{{end}}
```

Comment:
You don't need `if` here, just keep the `{{ toYaml .Values.labels | indent 8 }}` and add `labels: {}` and `annotations: {}` in values file. Same for others.

Btw, labels are usually not configurable, only annotations if you look at official charts, but we can keep it here.